src/coord/src/coord.rs

```diff
@@ -3053,6 +3035,9 @@ impl Coordinator {
                                 -1,
                             )
                             .await;
+                            if let Some(tables) = &mut self.persisted_tables {
```

Comment:
Ok so ```DROP DATABASE``` is not simply an iteration over the objects dropping them individually? I would expect that the loop that is dropping the individual tables would cause their persisted representations to be removed.

If so, understood, I will make sure that I have tests that call  ```DROP DATABASE```  explicitly.